src/blocks/stars.css

```
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star :last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(1),
```

Maybe better to write like this.
```suggestion
.stars--1 :nth-child(-n + 1),
```

---

In my opinion, both options are correct, but I would like to hear the opinion of the mentor